### DISCUSSION TITLE
Filament end sensor change from low to high

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [Jyers/Marlin#1438] [Q&A]
Hi

I made a filament end sensor with a z-stop, unfortunately the logic is the reverse. When pressed, reports a problem with the filament. I read that it is activated with a high state. Is there any simple method to go from low to high? Is the whole Jyers configuration available somewhere so that I can only change this one parameter and the rest will stay the same?

Is there any other way to do it without compiling the software?

**Comment by jetel73** [comment]
Hello, it should be easier the hard way... :)
You probably already know this but if not... Those switches are 2 states High and Low simultaneously the signal depends on the pins you choose. It is the reason for the 3 pins. If you keep the left and right and avoid the middle one you should send the right signal.
I've made exactly the same run out sensor and it works great.
